src/java/test/org/apache/zookeeper/server/quorum/QuorumPeerMainTest.java

```diff
@@ -452,7 +452,7 @@ public void testElectionFraud() throws IOException, InterruptedException {
         Assert.assertTrue("falseLeader never rejoins the quorum", foundFollowing);
     }
 
-    private void waitForOne(ZooKeeper zk, States state) throws InterruptedException {
```

Comment:
this are in the other change, so we are going to have to re-resolve depending on which one goes in.

Reply:
@breed This is based on current master branch, we don't have dependency as far as I know, so it should be ready to go in. 

(sorry for the lately reply, just came back from vocation)